CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
-## [*Unreleased*](https://github.com/pbrisbin/bugsnag-haskell/compare/v0.0.3.0...master)
+## [_Unreleased_](https://github.com/pbrisbin/bugsnag-haskell/compare/v0.0.3.0...master)
 
-None
+- Fixed serialization of headers in BugsnagRequest. Previously, headers were a
+  list of:
+
+```
+[(CI ByteString, ByteString)]
+```
+
+so they serialized as an array, but Bugsnag only accepts objects for headers.
+
+To solve this, the newtype `BugsnagRequestHeaders` now wraps the headers and has
+a correct ToJSON instance.
+
+This also allowed removing the orphan ToJSON instance for CI (from the
+`case-insensitive` package).
```

Comment:
Can you shift this content two spaces, so it renders as part of the list element?